src/main/java/software/amazon/nio/spi/s3/S3FileSystemProvider.java

```diff
@@ -74,7 +75,7 @@ public class S3FileSystemProvider extends FileSystemProvider {
     /**
      * Constant for the S3 scheme "s3"
      */
-    public static final String SCHEME = "s3";
+    static final String SCHEME = "s3";
```

Comment:
Consider moving to `Constants`

Reply:
https://github.com/awslabs/aws-java-nio-spi-for-s3/pull/227#discussion_r1376624095